app/code/core/Mage/Bundle/Block/Sales/Order/Items/Renderer.php

```diff
@@ -140,7 +140,7 @@ public function getValueHtml($item)
     }
 
     /**
-     * Getting all available childs for Invoice, Shipmen or Creditmemo item
+     * Getting all available children for Invoice, Shipmen or Credit memo item
```

Comment:
I noticed Shipmen should be Shipment.
```suggestion
     * Getting all available children for Invoice, Shipment or Credit Memo item
```